frontend/pulse3d/src/components/uploads/TableRow.js

```diff
@@ -102,6 +102,8 @@ export default function Row({
   };
 
   const formatAnalysisParamColumn = (param, analyzedFile, value) => {
+    // Tanner (8/23/22): older analyses did not store the analysis params in the metadata of their DB entries,
```

Comment:
I think this should be moved down to line 184 since that's the condition I had originally intended for this comment to explain